draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2202,12 +2202,58 @@ uCDN-A                         dCDN-B                         dCDN-C
 	</texttable>
 	</section>
 
+	<section title="URL Type" anchor="url-types">
+	<t>
+	This type is used by uCDN to indicate how to interpret URLs referenced by trigger specs that use URLs, such as 
+		<xref target="urls-spec"/>, <xref target="uri-pattern-match-spec"/>, <xref target="uri-regex-match-spec"/> and <xref target="objectlist-spec"/>.
+	</t>
+	<t>
+	One option for uCDN to use in Trigger Commands is published URLs, that are used by end users.
+	When using this URL type, uCDN MUST provide configuration metadata objects related to these URLs before creating a trigger option referencing these URLs. When this is not the case, dCDN MUST return error code "emeta".
+	dCDN 
+	</t>
+	<t> 
+		When processing published URLs in "preposition" Trigger Action, dCDN MUST invoke processing of metadata objects, it would have involved in content acquisition to satisfy an end user request, e.g. SourceMetaData (see Section 4.2.1 of <xref target="RFC8006"/>.
+	</t>
+
+	<t>
+		Another type of URL in common use is private URL based on cache keys that are dynamically 
```

Comment:
Reword: Another type of URL in common use is the private URL, which is based on cache keys..."

Reply:
Accepted